core/embed/prodtest/main.c

```diff
@@ -761,7 +943,10 @@ int main(void) {
 
     } else if (startswith(line, "FIRMWARE VERSION")) {
       test_firmware_version();
-
+    } else if (startswith(line, "BOOTLOADER VERSION")) {
+      test_bootloader_version(bootloader_version);
```

Comment:
It would be better to be consistent with the other `test_` functions and have `test_bootloader_version` and `test_boardloader_version` without any arguments

Reply:
the problem here is that due to MPU configuration bootloader header is unreadable at the time this command is called, so i need to parse the header before the MPU is configured and store it in the variable. I could avoid the parameter by using global variable but i don't think its necessarily better. 

Boardloader version test would work without param, but at this at this point the consistency is lost anyway i don't think it matters much